website/docs/docs/deploy/source-freshness.md

```diff
@@ -12,7 +12,14 @@ dbt Cloud provides a helpful interface around dbt's [source data freshness](/doc
 
 ### Enabling source freshness snapshots
 
-First, make sure to configure your sources to [snapshot freshness information](/docs/build/sources#snapshotting-source-data-freshness).
+`dbt build` does _not_ include source freshness checks when it builds and tests resources in your DAG. As such, here's a common pattern for defining jobs:
+- `dbt build` as the run step
```

Comment:
I'd suggest removing the <tabItems> on this page too. It would be nice to link to a section from this bullet.